Cargo.toml

```diff
@@ -43,13 +44,17 @@ ethers = { version = "2.0", features = ["solc"] }
 futures = "0.3"
 hotshot = { git = "https://github.com/EspressoSystems/hotshot", tag = "0.5.28" }
 hotshot-orchestrator = { git = "https://github.com/EspressoSystems/hotshot", tag = "0.5.28" }
-hotshot-query-service = { git = "https://github.com/EspressoSystems/hotshot-query-service", tag = "0.1.0" }
+#hotshot-query-service = { git = "https://github.com/EspressoSystems/hotshot-query-service", tag = "0.1.0" }
```

Comment:
to be fixed before making non-draft... including removing all commented out lines.